### PR TITLE
Upgrade to Gradle 4.4.1 and use JCenter first to resolve plugins

### PR DESCRIPTION
**Pull request by lptr**
This works around https://github.com/Kotlin/dokka/issues/146

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,3 +1,12 @@
+// Temporary worakround for https://github.com/Kotlin/dokka/issues/146
+// Must put jcenter() fist to resolve the correct artifacts for Dokka
```
Comment (vanniktech) [inline settings.gradle]: There's a small typo regarding first

Comment (lptr) [reply settings.gradle]: Thanks, fixed.

**Review by vanniktech — APPROVED** [review]

File: settings.gradle [hunk 2]
```diff
@@ -1,3 +1,12 @@
+// Temporary worakround for https://github.com/Kotlin/dokka/issues/146
+// Must put jcenter() first to resolve the correct artifacts for Dokka
```
Comment (artem-zinnatullin) [inline settings.gradle:2]: oh god

**Review by artem-zinnatullin — APPROVED** [review]
lgtm, looks like gradle-wrapper.jar haven't changed, but I'll update it separately if needed

**Comment by artem-zinnatullin** [comment]
Thanks!

**Pull request merged**